extension-compatibility.json

```diff
@@ -1,4 +1,9 @@
 {
+  "com.civibridge.quickmenu": {
+    "obsolete": "5.24",
+    "disable": true,
```

Comment:
@colemanw Wasn't aware of "obsolete" capability. Kinda cool. Is this documented somewhere?

Reply:
No it's fairly new. So far its being used for extensions whose features have been migrated into core (Api4, KAM, ExportUI, etc.). Adding an extension to this list will:

1. Automatically disable/uninstall it during the upgrade.
2. Not allow it to be reenabled; label it "obsolete" in the UI.
3. Resolve any dependencies (so an extension that declares a dependency on Api4 will still work)

Let me see if I can find a spot for that in the dev docs pages...

Reply:
https://github.com/civicrm/civicrm-dev-docs/pull/764